src/handlers/warehouse.ts

```diff
@@ -54,13 +80,22 @@ export const warehouseHandlers: NatsHandler[] = [
             const operation = await freeze_item(warehouseContract, collectible);
 
             await operation.confirmation(1, 1);
+
+            client.publish(
```

Comment:
same comment as above, let's put the client.publish in a condition and just publish the item_id without the text around it